Sources/AWSLambdaRuntimeCore/Lambda+String.swift

```diff
@@ -13,25 +13,46 @@
 //===----------------------------------------------------------------------===//
 import NIOCore
 
-extension EventLoopLambdaHandler where Event == String {
+// MARK: - LambdaHandler String support
+
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
```

Comment:
Do we just want to require macOS 12 and friends in the `Package.swift`? Will be a breaking change after 1.0...

Reply:
sure

Reply:
This makes sense to me since most of the functionality I look to use is barred behind this anyways